flink-test-utils-parent/flink-test-utils-junit/src/main/java/org/apache/flink/util/DockerImageVersions.java

```diff
@@ -40,7 +40,7 @@
 
     public static final String KINESALITE = "instructure/kinesalite:latest";
 
-    public static final String LOCALSTACK = "localstack/localstack:latest";
+    public static final String LOCALSTACK = "localstack/localstack:0.13.3";
```

Comment:
@dannycranmer This can backfire at any time (and it already did, but the docker cache hides it on CI). Same should apply for the kinesis above.

Reply:
Ack, thanks for the spot. We will update the kinesalite image